build.sh

```diff
@@ -0,0 +1,12 @@
+sudo steamos-readonly disable
+curl -s https://raw.githubusercontent.com/89luca89/distrobox/main/install | sudo sh -s -- --prefix /usr
+sudo steamos-readonly enable
+xhost +si:localuser:$USER >/dev/null
+distrobox create && distrobox enter
+curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+sudo dnf install pkg-config openssl-devel
+sudo dnf install llvm-devel
+sudo dnf install gcc-c++.x86_64
```

Comment:
We don't need to install Winetricks? Does it use the Wintricks installed directly from the host?

Reply:
I didn't need it in order to compile.

Reply:
I think so, but you need it in order to install the fonts and dependecies for the game